pycloudlib/azure/cloud.py

```diff
@@ -887,7 +888,14 @@ def _retrive_instance_ip(self, instance):
 
         return self._retrieve_ip_from_network_interface(nic=instance_nic)
 
-    def get_instance(self, instance_id, search_all=False):
+    def get_instance(
+        self,
+        instance_id,
+        search_all=False,
+        *,
+        username: Optional[str] = None,
+        **kwargs,
```

Comment:
`kwargs` is not passed anywhere. If this is needed for signature compatibility, do you think we should add a comment in the docstring to mention why?